doc/source/release-notes/Percona-Server-5.7.27-30.rst

```diff
@@ -0,0 +1,128 @@
+.. rn:: 5.7.27-30
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on
+|date|. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.27-30/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+	
+This release is based on `MySQL 5.7.27
+<https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-27.html>`_ and includes
+all the bug fixes in it. |Percona Server| |release| is now the current GA
+(Generally Available) release in the 5.7 series.
+	
+All software developed by Percona is open-source and free.
+
+In this release, |Percona Server| introduces the variable
```

Comment:
New Features header?

Reply:
Dropped the paragraph.

Reply:
Checked with George - no new features needed.